aireview/README.md

```diff
@@ -1,2 +1,54 @@
 # AIReview
-Pull request review agent
\ No newline at end of file
+Pull request review agent
```

Comment:
**documentation**: Deployment section should specify the expected deployment environments and any specific configuration needed for each environment